### PR TITLE
Fix log format of web to be more splunky

### PR DESCRIPTION
**Pull request by mouadino**
Add also a unit for duration for us poor human that cant' compile
seconds milliseconds faster :)

### CODE REVIEW

File: lymph/core/rpc.py [hunk 1]
```diff
@@ -180,7 +180,7 @@ def dispatch_request(self, msg):
                 logger.exception('failed to send automatic NACK')
         finally:
             elapsed = (time.time() - start) * (10 ** 3)
-            logger.log(loglevel, '%s duration=%.3f', msg.subject, elapsed)
+            logger.log(loglevel, '%s duration=%.3f (seconds)', msg.subject, elapsed)
```
Comment (vpikulik) [inline lymph/core/rpc.py]: 'subject=%s duration=%.3f (seconds)' it should be better for splunk

Comment (mamachanko) [inline lymph/core/rpc.py]: quote it if in doubt

Comment (mouadino) [inline lymph/core/rpc.py]: quote what ?

**Comment by vpikulik** [comment]
:+1:

**Pull request merged**